src/Stratis.Bitcoin/P2P/Protocol/Payloads/GetProvenHeadersPayload.cs

```diff
@@ -0,0 +1,52 @@
+﻿using NBitcoin;
+using NBitcoin.Protocol;
+
+namespace Stratis.Bitcoin.P2P.Protocol.Payloads
+{
+    /// <summary>
+    /// Ask for proven block headers that happened since BlockLocator.
+    /// </summary>
+    [Payload("getprovhdr")]
+    public class GetProvenHeadersPayload : Payload
+    {
+        private uint version = (uint)ProtocolVersion.PROTOCOL_VERSION;
```

Comment:
Do we not want to bump this up to a higher version for proven headers? Or is that going to be done separately (I think there was a task for that)?

Reply:
Yeah, waiting for PR from @ferdeen. Let me put a note not to merge yet

Reply:
Not sure if we need version here. Usually it's sent in the `VersionPayload` during the handshake

Reply:
@noescape00 should we remove it then?